request.js

```diff
@@ -50,6 +50,15 @@ SuiteRequest.prototype = {
   },
 
 
+  delete: function(path) {
+    var options = this._getOptionsFor('DELETE', path);
+    var signedOptions = this._signRequest(options, '');
```

Comment:
There are duplications in this file and this commit only make it worst. If you could refactor the signing, logging and sending part of the these methods, please do it.